### PR TITLE
Test configure before install flake

### PR DESCRIPTION
**Pull request by jelly**
It turns out that we clicked on the memory modal button when the memory usage in our UI is still 150 MiB and the memory modal either does not update or updates too slow to get the new 130 MiB value.

### CODE REVIEW

File: test/check-machines-create [hunk 1]
```diff
@@ -2060,6 +2060,7 @@ vnc_password= "{vnc_passwd}"
 
         # Check configuration changes survived installation
         # Check memory settings have persisted
+        b.wait_in_text("#vm-subVmTest1-memory-count", "130.0 MiB")
```
Comment (martinpitt) [inline test/check-machines-create:2063]: ITYM `#vm-VmNotInstalled-memory-count` ?

Comment (jelly) [reply test/check-machines-create]: Yes..

**Comment by jelly** [comment]
Not sure what is going on with fedora-39/firefox

**Comment by martinpitt** [comment]
The [rawhide failures](https://artifacts.dev.testing-farm.io/0a63986c-a4fa-4872-9e67-d1f8abdd496d/) have been going on for a few days, that's the next yak to shave.

**Review by martinpitt — APPROVED** [review]
Thanks for tracking this down!

**Pull request merged**